github/repos_prereceive_hooks.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// PreReceiveHook represents a GitHub pre-receive hook for a repository.
+type PreReceiveHook struct {
+	ID          *int64  `json:"id,omitempty"`
+	Name        *string `json:"name,omitempty"`
+	Enforcement *string `json:"enforcement,omitempty"`
+	ConfigURL   *string `json:"configuration_url,omitempty"`
+}
+
+func (h PreReceiveHook) String() string {
```

Comment:
Go style nit: `func (p PreReceiveHook)...`
s/h/p/ to match the receiver variable with the start of the name of its type...
although I see that this Go style recommendation is not followed consistently throughout this repo, so you can ignore this comment safely.  I'll leave it anyway in case someone wants to clean up all receivers in this repo. :smile: